src/chips/sam4l/usart.rs

```diff
@@ -176,7 +176,8 @@ impl USART {
 }
 
 impl DMAClient for USART {
-    fn xfer_done(&mut self) {
+#[allow(unused_variables)]
```

Comment:
FYI: an alternative to `allow(unused_variables)` is to prefix the variable name with an underscore  (e.g. `_pid`). I think it's ever so slightly more future proof.